src/main/java/com/hida/Minter.java

```diff
@@ -0,0 +1,475 @@
+package com.hida;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+import org.codehaus.jackson.map.ObjectMapper;
+
+/**
+ * Minter class creates formatted ids in json using a variety of parameters.
+ *
+ * @author Brittany Cruz
+ * @author lruffin
+ */
+public class Minter {
+
+    private final static DatabaseManager DATABASE_MANAGER
+            = new DatabaseManager();
+    private final String CHAR_MAP;
+    private final HashMap<Character, String> BASE_MAP = new HashMap();
+
+    // fields    
+    private final String NAAN = "/:ark/70111/";
+    private final String PREPEND;

```

Comment:
style guide says only constants should be in all caps.  I realize that the guide is a little confusing because it says 

> (final variables)

but that's really if you declare and assign it in the same line.  `final` variables that are set in the constructor and are based on the constructor paramaters really can't be considered "constants"